starknet_py/net/client_models.py

```diff
@@ -173,6 +176,24 @@ class TransactionStatus(Enum):
     REJECTED = "REJECTED"
 
 
+@dataclass
+class BuiltinInstanceCounter:
+    pedersen_builtin: Optional[int] = None
+    range_check_builtin: Optional[int] = None
+    bitwise_builtin: Optional[int] = None
+    output_builtin: Optional[int] = None
+    ecdsa_builtin: Optional[int] = None
+    ec_op_builtin: Optional[int] = None
+    poseidon_builtin: Optional[int] = None
+
+
+@dataclass
+class ExecutionResources:
+    n_steps: int
+    n_memory_holes: int
+    builtin_instance_counter: BuiltinInstanceCounter
+
+
```

Comment:
Let's not type this ourselves. It's a lot of internal stuff that could change and probably would be sufficient to make it just a dictionary.